packages/code-studio/src/styleguide/Inputs.tsx

```diff
@@ -235,25 +235,38 @@ function Inputs(): React.ReactElement {
         <div className="col">
           <div className="form-group">
             <h5>Selection Menu</h5>
-            <select defaultValue="0" className="custom-select">
+            <Select
+              onChange={v => {
```

Comment:
I would break out a `noOp` function outside of the component as minor optimization to not cause re-renders

Reply:
For the styleguide I think this is fine.